### PR TITLE
Kebo/collection about

### PR DESCRIPTION
**Pull request by kristianeboe**
Adds
- collection about page
- collection module to centralize getServerSideProps for pages in that folder
- small readme update

### CODE REVIEW

File: src/pages/collections/[address]/nfts.tsx [hunk 1]
```diff
@@ -17,32 +17,9 @@ import { useRouter } from 'next/router';
...
-    };
-  }
-
-  return {
-    props: {
-      collection,
-      ...i18n,
-    },
-  };
-}
```
Comment (kespinola) [inline src/pages/collections/[address]/nfts.tsx:45]: I recommend against generalizing the gSS call at this time. Its a spot that is best for a middleware pattern. I advocate for keeping the duplication for now and revisiting drying up the gSS calls in the near future.

File: src/pages/collections/[address]/about.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        {props.creator.twitterHandle || shortenAddress(props.creator.address)}
+      </div>
+      <Button
+        className="ml-auto"
+        icon={<PlusIcon width={14} height={14} />}
+        size={ButtonSize.Small}
+      >
+        {t('follow', { ns: 'common' })}
+      </Button>
+    </div>
```
Comment (kespinola) [inline src/pages/collections/[address]/about.tsx:54]: I'm for keeping this inline with the existing page. Its not a large one and this won't be used else where.

File: src/pages/collections/[address]/about.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+  return (
+    <div className="flex w-full items-center">
+      <img
+        className="h-8 w-8 rounded-full"
+        src={props.creator.previewImage || addressAvatar(props.creator.address)}
+        alt="creator profile picture"
+      />
+      <div className="ml-4 text-base font-medium">
+        {props.creator.twitterHandle || shortenAddress(props.creator.address)}
```
Comment (kespinola) [inline src/pages/collections/[address]/about.tsx:45]: I created a client side field on wallet called displayName

https://github.com/holaplex/dao-marketplace/blob/main/src/client.ts#L183

In your query request

```graphql
displayName @client
previewImage @client
```

This defaulting is done at the data layer now.

File: src/pages/collections/[address]/about.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+function CreatorRow(props: { creator: NftCreator }) {
+  const { t } = useTranslation(['collection', 'common']);
+
+  return (
+    <div className="flex w-full items-center">
+      <img
+        className="h-8 w-8 rounded-full"
+        src={props.creator.previewImage || addressAvatar(props.creator.address)}
```
Comment (kespinola) [inline src/pages/collections/[address]/about.tsx:41]:
```suggestion
        src={props.creator.previewImage}
```

previewImage takes care of defaulting to addressAvatar

File: src/pages/collections/[address]/about.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+export const getServerSideProps = getCollectionServerSideProps;
+
+export default function CollectionAboutPage(props: { collection: Collection }) {
+  const { t } = useTranslation(['collection', 'common']);
+
+  return (
+    <div className="  py-6 px-10 text-white">
+      <section className="space-y-4">
+        <h2 className="font-semibold">{t('aboutPage.collectionDescription')}</h2>
```
Comment (kespinola) [inline src/pages/collections/[address]/about.tsx:18]:
```suggestion
        <h2 className="font-semibold">{t('descriptionTitle')}</h2>
```

??

File: src/pages/collections/[address]/about.tsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+        <p className="text-gray-300">{props.collection.nft.description}</p>
+      </section>
+      <div className="my-6 border border-gray-800" />
+      <section className="space-y-4">
+        <h2 className="font-semibold">{t('aboutPage.creators')}</h2>
```
Comment (kespinola) [inline src/pages/collections/[address]/about.tsx:23]:
```suggestion
        <h2 className="font-semibold">{t('creatorsTitle')}</h2>
```

??

File: src/modules/collections.ts [hunk 7]
```diff
@@ -0,0 +1,31 @@
+import { GetServerSidePropsContext } from 'next';
+import { SSRConfig } from 'next-i18next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import client from '../client';
+import { CollectionQuery } from '../queries/collection.graphql';
+import { Collection } from '../types';
+
+export async function getCollectionServerSideProps({ locale, params }: GetServerSidePropsContext) {
```
Comment (kespinola) [inline src/modules/collections.ts:8]: Noted in another comment that like to hold off in consolidating the gss calls for the time being.

File: README.md [hunk 8]
```diff
@@ -1,16 +1,25 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
 ## Getting Started
+First, install dependencies
+```bash
+yarn install
```
Comment (kespinola) [inline README.md:6]: Looking to drop yarn for this project and just rock npm.

```
npm install
```

File: yarn.lock [hunk 9]
```diff
@@ -3937,6 +3937,11 @@ react-dom@18.2.0:
     loose-envify "^1.1.0"
```
Comment (kespinola) [inline yarn.lock:3937]: delete. going with npm for this project.

**Review by kespinola — CHANGES_REQUESTED** [review]
requests up for discussion.

File: src/pages/collections/[address]/about.tsx (hunk 2 above)
Comment (kristianeboe) [reply src/pages/collections/[address]/about.tsx:54]: This UI pattern is actually present in some other places like the Following/Followers modal etc. I figured I'd just keep it in this page and then generalize it a bit when I got to that section.

Comment (kristianeboe) [reply src/pages/collections/[address]/about.tsx:54]: It was called FollowItem before for some reason, a better name is probably WalletProfileRow, or something similar...

**Pull request merged**